src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,16 @@
 package mate.academy
 
+import mate.academy.model.Figure
+import mate.academy.service.FigureSupplier
+
+private const val FIGURES_AMOUNT = 8
+
 fun main() {
-    // draw all figures here
+    val figureSupplier = FigureSupplier()
+    val figures: Array<Figure> = Array(FIGURES_AMOUNT) {
+        i -> if (i < FIGURES_AMOUNT / 2) figureSupplier.getRandomFigure()
+    else figureSupplier.getDefaultFigure()
+    }
+    
+    figures.forEach { f -> f.draw() }
```

Comment:
It is better to use more meaningful naming for loop items in similar cases, otherwise there is no difference with 'it' parameter. Also, you may just use 'it' to simplify expression in such non-complex functionality